app/models/world_location.rb

```diff
@@ -104,7 +104,7 @@ def to_s
   end
 
   def self.all_by_type
-    ordered_by_name.group_by(&:world_location_type)
+    ordered_by_name.in_order_of(:world_location_type, %w[world_location international_delegation]).group_by(&:world_location_type)
```

Comment:
Question out of interest more than anything - is it possible to re-use the enum here to define the ordering?

Reply:
I did wonder if we could do that, but decided against it as any future additions to the enum may not necessarily appear in the order we want them to be listed on the website.